tests/data/parsers/test_parser_depth_average_rule.py

```diff
@@ -0,0 +1,73 @@
+# This file is part of D-EcoImpact
+# Copyright (C) 2022-2023 Stichting Deltares
```

Comment:
copy right is incorrect. Please check all (test) files

Reply:
check (60files updated to 2022-2024)